clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -53,39 +55,99 @@ const stubConfig = ({
           ? undefined
           : Object.assign(config.options, options),
     };
-    if (typeof options !== "string" && options?.geolocationApiUrl) {
-      cy.intercept("GET", options.geolocationApiUrl, {
+    if (
+      typeof updatedConfig.options !== "string" &&
+      updatedConfig.options?.geolocationApiUrl
+    ) {
```

Comment:
Why is this so conditional and we don't just always stub this API?

Likely deserves a explanatory comment at least

Reply:
you're right, we can always stub these APIs

Reply:
Added a comment - `we conditionally stub these APIs because we need the exact API urls, which can change or not even exist in the specific test case.`